### PR TITLE
Nk add puzzle element quiz

### PR DESCRIPTION
**Pull request by pcsmomo**
# Element Quiz (with Periodic Table)

I have added 8 sets of quizzes and answers.  
We could disable them if we do not need as many :)

## The SQL to initiate the database for this puzzle

```sql
INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_general', 'Au', 1, '/element', '');

INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_xas', 'Fe', 1, '/element_xas', '');

INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_mex', 'Cu', 1, '/element_mex', '');

INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_ads', 'Ge', 1, '/element_ads', '');

INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_bsx', 'N', 1, '/element_bsx', '');

INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_mct', 'Ca', 1, '/element_mct', '');

INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_mx', 'C', 1, '/element_mx', '');

INSERT INTO puzzle
(name, answer, active, location, notes)
VALUES('element_pd', 'Si', 1, '/element_pd', '');
```

## To add more quizzes/answers and beamlines
- `openday_scavenger/puzzles/element/static/data/questions_answers.py`: Quiz database
- `openday_scavenger/puzzles/element/static/data/questions_beamline.py`: Apply beamline
- `openday_scavenger/puzzles/__init__.py`: Router

### CODE REVIEW

File: openday_scavenger/puzzles/element/views.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    # choose a random question
+    question = random.choice(questions)
+
+    return templates.TemplateResponse(
+        request=request,
+        name="index.html",
+        context={
+            "puzzle": puzzle_name,
+            "visitor": visitor.uid,
```
Comment (stmudie) [inline openday_scavenger/puzzles/element/views.py]: The visitor entry for the context has been deprecated. The UID is now retrieved automatically from the cookie.

File: openday_scavenger/puzzles/element/static/sql/initiate.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
+INSERT INTO puzzle
```
Comment (stmudie) [inline openday_scavenger/puzzles/element/static/sql/initiate.sql]: Can you please remove this file from the PR? We don't need it in main.

**Review by stmudie — COMMENTED** [review]
This PR is amazing! I've just left a couple of small comments to address. Thanks!!!

File: openday_scavenger/puzzles/element/views.py (hunk 1 above)
Comment (pcsmomo) [reply openday_scavenger/puzzles/element/views.py]: Thank you for the review! Yes, it is removed now!

**Comment by pcsmomo** [comment]
Thank you for reviewing, @stmudie , 
Changes have been committed!

**Review by stmudie — APPROVED** [review]
Fantastic puzzle. :)

**Pull request merged**